ReactWindows/ReactNative/Accessibility/AccessibilityHelper.cs

```diff
@@ -407,8 +473,11 @@ private static void UpdateName(UIElement element)
             var peer = FrameworkElementAutomationPeer.FromElement(element);
             if (IsInGenerativeState(element))
             {
-                var generatedName = GenerateNameFromUpdated(peer);
-                AutomationProperties.SetName(element, generatedName);
+                if (AutomationProperties.GetAccessibilityView(element) != AccessibilityView.Raw)
+                {
```

Comment:
Why is this code specific to "generative" state?
We can trigger a SetName in line 491 just by setting a spurious "Label" on a non-generative element...

Reply:
Discussed offline. This is specific for "generated" because in this case 'we' (the framework) is responsible for correctness.  In line 491, the user (of the control) sets the name and it is up to them to make sure that it is correct.

---
In reply to: [173237324](https://github.com/Microsoft/react-native-windows/pull/1698#discussion_r173237324) [](ancestors = 173237324)